test/datomish/tofinoish_test.cljc

```diff
@@ -460,3 +466,24 @@
         (is (= apricot-url "http://example.com/apricots/1"))
         (is (= apricot-title "A page about apricots."))
         (is (= apricot-excerpt "")))))))
+
+(deftest-db test-fulltext-set-attribute conn
+  (<? (d/<transact! conn tofino-schema))
+  (<? (d/<transact! conn
+                    [{:db/id             999
```

Comment:
Do we hit the case where `:a` is `:db/fulltext true`, the value would match, but it's excluded from the search?  I don't think we do here.